packages/react-slider/src/components/Slider/Spec.md

```diff
@@ -1,146 +1,42 @@
-# @fluentui/react-slider Draft Component Spec
```

Comment:
The spec for Slider isn't in the root folder, unlike other component packages -- maybe better to move it there?